docs/devel_doc/container_orchestration.md

```
@@ -166,19 +166,19 @@ This ensures the OGX container is always cleaned up, even if the service crashes
 
 **Stop the container (keeps container for inspection):**
 ```bash
-make stop-llama-stack-container
+make stop-ogx-container
 ```
 
 **Remove the container (saves logs first):**
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)